components/elm/src/external_models/emi/src/elm_stub/utils/clm_varctl.F90

```diff
@@ -372,7 +372,7 @@ module clm_varctl
   logical, public :: pf_tmode     = .false.                 ! switch for 'T' mode coupling (will be updated in interface)
   logical, public :: pf_frzmode   = .false.                 ! switch for 'freezing' mode availablity in PF-thmode (will be updated in interface)
   logical, public :: initth_pf2clm= .false.                 ! switch for initializing CLM TH states from pflotran
```

Comment:
Rename initth_pf2clm?

Reply:
[Same as the previous comment](https://github.com/E3SM-Project/E3SM/pull/3842#discussion_r535744540)